src/katello/tracer/dnf.py

```diff
@@ -0,0 +1,91 @@
+import os
+import subprocess
+import psutil
+from katello.tracer.SystemdDbus import SystemdDbus
+
+STATIC_SERVICES = [
+    "systemd",
+    "dbus",
+]
+
+REBOOT_HELPER= 'You will have to reboot your computer'
+SESSION_HELPER = 'You will have to log out & log in again'
+
+class DnfTracerApp:
```

Comment:
I wonder why there's no common class for the results. Aren't they all the same for the various implementations?

Reply:
Oh, thats true. It would be better to have a generic TracerApp and re-use this for zypper, deb and dnf. We can restructure this after this PR.